### ISSUE TITLE
Issue with subdomain on different IP or account?

### ISSUE THREAD

**Issue by BeZazz** [Prajithp/letsencrypt-cpanel#26]
I set up a subomain via cPanel and installed letsencrypt via cPanel.
(no issues)

I uninstalled the subomain.

I then used the same name and created an account via WHM.
cPanel could not install the cert but WHM could.

**Comment by Prajithp** [comment]
What error message are you getting there? I couldn't able to recreate the issue on my stagging server.

**Comment by setroot** [comment]
I also did not encounter any errors.

**Comment by BeZazz** [comment]
I am guessing this is the same issue as
https://github.com/Prajithp/letsencrypt-cpanel/issues/21

**Comment by Prajithp** [comment]
@BeZazz is there any way to recreate this issue? In my system, I couldn't able to recreate.

 If possible, please ping me on prajithpalakkuda@gmail.com

**Comment by BeZazz** [comment]
From memory.

I created a subdomain (sub.domain.com) via cPanel on a domain with a dedicated IP.

I then decided I wanted to give the subdomain in its own account.

So I removed the subdomain via cPanel.

I then created the same domain (sub.domain.com) via WHM add account.

I was not able to add an SSL Cert via the plugin in cPanel (I couldn't find the logs)

I was able to use the plugin via WHM to add an SSL Cert.

**Comment by Prajithp** [comment]
@BeZazz Thank you will look into this.

**Comment by ceesco53** [comment]
I'm also experiencing this issue and it is the same as issue #21 .  Is there a way to validate a subdomain ssl when the domain already has a cert on a different IP?  Would it be worth trying to hack a PR for this module or is the issue more complicated?

EDIT: I just noticed the error is looking for a www.subdomain.domain.com.  I don't have a www A record under the subdomain.  This might be my fault entirely.
EDIT: Nevermind.  The plugin worked for me when I added the www A record for my subdomain.  I was able to create a subdomain ssl key on a different IP while simultaneously on a different account.

**Comment by Prajithp** [comment]
@ceesco53 Yes, you must create a dns record for both www and non www address as this plugin completely depending on cPanel/WHM api, so if api returns www.sudoman.domain.com and it doesn't have valid A record then letsencrypt verification will definitely fail.
